components/vc-drawer/src/Drawer.js

```diff
@@ -142,7 +142,7 @@ const Drawer = defineComponent({
   methods: {
     domFocus() {
       if (this.dom) {
-        this.dom.focus();
+        this.dom.focus({ preventScroll: true });
```

Comment:
Is it better to change to `setTimeout` where domFocus is triggered?
https://github.com/vueComponent/ant-design-vue/blob/dd859500c2d8c58f1f12dd46ba3330cceee3bd30/components/vc-drawer/src/Drawer.js#L84

Reply:
I tried, it actually work. Given we found the root cause is calling `focus` on invisible element without preventScroll, we can fix it by
- pass `preventScroll` to `focus`
- or guarantee element appeared in viewport before `focus` called, so yes `setTimeout` would do the job, but I'm not sure will it cause any other side effects.

And I personally always consider `setTimeout` to be the last resort to resolve issues if you have any other options.

Reply:
you are right.  but preventScroll will cause breaking change( we need scrollIntoView).
For 2.x, we can use setTimeout resolve it .  we will refactor render logic in v3.x.